app/models/commit.js

```diff
@@ -49,9 +49,11 @@ export default Model.extend({
     }
   ),
 
-  githubUrl: computed('build.repo.slug', 'sha', function () {
-    let slug = this.get('build.repo.slug');
-    let sha = this.sha;
-    return this.externalLinks.githubCommit(slug, sha);
+  githubUrl: computed('build.repo.{slug,vcsType}', 'sha', function () {
```

Comment:
Shouldn't we change **githubUrl** name to something more general? Even not only git*Url, as not all version control systems are gits. Changing name of this function will affect builds-item.hbs and dashboard-row.hbs

Reply:
Will do, thanks!